### PR TITLE
Improved customisable homepage and added badge

### PR DESCRIPTION
**Pull request by arcbtc**
image

### CODE REVIEW

**Review by talvasconcelos — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -182,6 +182,9 @@ LNBITS_DEFAULT_WALLET_NAME="LNbits wallet"
 # LNBITS_AD_SPACE_TITLE="Supported by"
 # csv ad space, format "<url>;<img-light>;<img-dark>, <url>;<img-light>;<img-dark>", extensions can choose to honor
 # LNBITS_AD_SPACE="https://shop.lnbits.com/;https://raw.githubusercontent.com/lnbits/lnbits/main/lnbits/static/images/lnbits-shop-light.png;https://raw.githubusercontent.com/lnbits/lnbits/main/lnbits/static/images/lnbits-shop-dark.png"
+# LNBITS_HP_ELEMENTS=true # if set to true, the ad space will be displayed on the home page
```
Comment (motorina0) [inline .env.example:185]: what does `HP` stands for?

File: lnbits/helpers.py [hunk 2]
```diff
@@ -58,6 +58,9 @@ def template_renderer(additional_folders: Optional[List] = None) -> Jinja2Templa
     t.env.globals["LNBITS_DENOMINATION"] = settings.lnbits_denomination
     t.env.globals["SITE_TAGLINE"] = settings.lnbits_site_tagline
     t.env.globals["SITE_DESCRIPTION"] = settings.lnbits_site_description
+    t.env.globals["LNIBITS_HP_ELEMENTS"] = settings.lnbits_hp_elements
+    t.env.globals["LNIBITS_CUSTOM_BADGE"] = settings.lnbits_custom_badge
+    t.env.globals["LNIBITS_CUSTOM_BADGE_COLOR"] = settings.lnbits_custom_badge_color
```
Comment (motorina0) [inline lnbits/helpers.py:63]: All these three properties have a typo (`LNIBITS` -> `LNBITS`).
```suggestion
    t.env.globals["LNBITS_HP_ELEMENTS"] = settings.lnbits_hp_elements
    t.env.globals["LNBITS_CUSTOM_BADGE"] = settings.lnbits_custom_badge
    t.env.globals["LNBITS_CUSTOM_BADGE_COLOR"] = settings.lnbits_custom_badge_color
```

File: lnbits/core/templates/core/index.html [hunk 3]
```diff
@@ -8,12 +8,16 @@
   ></div>
   <div v-else class="col-12 col-md-4 col-lg-4 q-gutter-y-md">
     <div class="gt-sm">
-      <h3 class="q-my-none" v-if="'{{SITE_TITLE}}' == 'LNbits'">
+      <h3 class="q-my-none" v-if="'{{LNIBITS_HP_ELEMENTS}}' == 'True'">
```
Comment (motorina0) [inline lnbits/core/templates/core/index.html:11]:
```suggestion
      <h3 class="q-my-none" v-if="'{{LNBITS_HP_ELEMENTS}}' == 'True'">
```

File: lnbits/core/templates/core/index.html [hunk 4]
```diff
@@ -8,12 +8,16 @@
...
         {{SITE_TITLE}}
       </h3>
-      <h5 class="q-my-md" v-if="'{{SITE_TITLE}}' == 'LNbits'">
+      <h5 class="q-my-md" v-if="'{{LNIBITS_HP_ELEMENTS}}' == 'True'">
```
Comment (motorina0) [inline lnbits/core/templates/core/index.html:14]:
```suggestion
      <h5 class="q-my-md" v-if="'{{LNBITS_HP_ELEMENTS}}' == 'True'">
```

File: lnbits/core/templates/core/index.html [hunk 5]
```diff
@@ -8,12 +8,16 @@
...
         {{SITE_TAGLINE}}
       </h5>
+      <div
+        v-html="formatDescription"
+        v-if="'{{LNIBITS_HP_ELEMENTS}}' == 'True'"
```
Comment (motorina0) [inline lnbits/core/templates/core/index.html:19]:
```suggestion
        v-if="'{{LNBITS_HP_ELEMENTS}}' == 'True'"
```

File: .env.example [hunk 6]
```diff
@@ -182,6 +182,9 @@ LNBITS_DEFAULT_WALLET_NAME="LNbits wallet"
...
+# LNBITS_CUSTOM_BADGE="USE WITH CAUTION - LNbits wallet is still in BETA" 
+# LNBITS_CUSTOM_BADGE_COLOR="warning"
```
Comment (motorina0) [inline .env.example:187]: This property is too fine grained, it pollutes the settings namespace.
```suggestion
```

File: .env.example (hunk 1 above)
Comment (arcbtc) [reply .env.example:185]: Yes. It has a tip saying so

File: .env.example (hunk 6 above)
Comment (arcbtc) [reply .env.example:187]: Its important the badge color is definable

File: .env.example (hunk 1 above)
Comment (arcbtc) [reply .env.example]: Done

**Review by motorina0 — APPROVED** [review]

**Pull request merged**